src/Controls/StreetControls.js

```diff
@@ -63,6 +57,14 @@ class StreetControls extends FirstPersonControls {
         this.keyGoToNextPosition = 90;
         this.keyGoToPreviousPosition = 83;
         this.keySetCameraToCurrentPosition = 65;
+
+        // Raycaster used to pick objects
+        this.raycaster = new THREE.Raycaster();
+        // Tween is used to make smooth animations
+        this.tweenGroup = new TWEEN.Group();
+
+        // init surfaces used as helper for end user
+        this.initSurfaces();
```

Comment:
if `initSurfaces` is used to instantiate the helpers, you could make a private fonction, like this:
```js
initSurfacesHelper(this);
```

Reply:
yes good idea